base/utferror.jl

```diff
@@ -0,0 +1,30 @@
+# This file is a part of Julia. License is MIT: http://julialang.org/license
+
+##\brief      Error messages for Unicode / UTF support
```

Comment:
I remember only one person complaining, and a number more who seemed to approve... I can't even find the disapproving comment(s), could you point them out?